src/liedle/index.js

```diff
@@ -19,7 +19,9 @@ function start() {
   isEndGame = false;
   createGrid();
   resetGame();
-  handleInput();
```

Comment:
can you briefly explain why don't we need handleInput() here?

Reply:
`handleInput()` starts handlers which parse and respond to keypresses, we only want it to be run if we get a `modal-closed` event as by default the modal is open.

Reply:
`handleInput()` starts handlers which parse and respond to keypresses, we only want it to be run if we get a `modal-closed` event as by default the modal is open and we don't want to respond to keypresses.